Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -15,37 +16,70 @@ namespace Leap.Unity {
    */
   public class HandPool :
     HandFactory {
-
+    [SerializeField]
+    private bool EnforceHandedness;
     [SerializeField]
     private List<ModelPair> ModelCollection;
     [SerializeField]
     private List<ModelGroup> ModelPool;
-    public bool EnforceHandedness = false;
-    
+    public List<HandRepresentation> ActiveHandReps = new List<HandRepresentation>();
```

Comment:
This variable is still public and should not be, since it should not be able to be modified by outside forces.